libs/seiscomp/io/database.cpp

```diff
@@ -108,63 +109,21 @@ bool DatabaseInterface::connect(const char* con) {
 		return false;
 
 	_timeout = 0;
-	string connection = con;
-	string params;
-	size_t pos = connection.find('?');
-	if ( pos != string::npos ) {
-		params = connection.substr(pos+1);
-		connection.erase(connection.begin() + pos, connection.end());
-	}
-
-	vector<string> tokens;
-	string host;
-	if ( Core::split(tokens, connection.c_str(), "@") >= 2 ) {
-		string login = tokens[0];
-		host = tokens[1];
 
-		Seiscomp::Core::split(tokens, login.c_str(), ":");
-		if ( tokens.size() > 0 ) _user = tokens[0];
-		if ( tokens.size() > 1 ) _password = tokens[1];
-	}
-	else
-		host = tokens[0];
+	// Dummy scheme to allow parsing with Util::Url
+	Util::Url url(string("dummy://") + con);
 
-	size_t splitter = host.find_first_of('/');
-	if ( splitter != string::npos ) {
-		tokens.resize(2);
-		tokens[0] = host.substr(0, splitter);
-		tokens[1] = host.substr(splitter+1);
-	}
-	else
-		tokens[0] = host;
+	auto path = url.path();
+	if (path[0] == '/') path = path.substr(1);
```

Comment:
Can you please add a space after the bracket as recommended in [SeisComP Coding Conventions](https://docs.gempa.de/seiscomp/current/base/coding-conventions.html)
Also we tend to follow [Clang's readability-braces-around-statements](https://releases.llvm.org/19.1.0/tools/clang/tools/extra/docs/clang-tidy/checks/readability/braces-around-statements.html) when writing new code:

```C++
if ( path[0] == '/' ) {
    path = path.substr(1);
}
```